source/includes/api/_query.md

```diff
@@ -478,37 +640,68 @@ You can read more about the limitations of using SQL with document-based dataset
 SQL-like queries can be employed for accessing data stored in Rasdaman datasets. Subsets on the original axes of the data may be provided in the WHERE statement. So far, only operations that result in a single scalar can be obtained from Rasdaman - averages, minimums, maximums.
 
 ```shell
-curl -XGET https://api.resourcewatch.org/v1/query?sql=SELECT avg(Green) from 18c0b71d-2f55-4a45-9e5b-c35db3ebfe94 where Lat > 0 and  Lat < 45 \
+curl -XGET 'https://api.resourcewatch.org/v1/query?sql=SELECT avg(Green) from 18c0b71d-2f55-4a45-9e5b-c35db3ebfe94 where Lat > 0 and  Lat < 45' \
   -H 'Content-Type: application/json' \
   -H 'Authorization: Bearer <token>'
 ```
 
-### NEX-GDDP datasets
-
-A SQL wrapper is offered for accessing the NASA NEX-GDDP dataset with sql-like statements. The API stores calculated indexes over the original data, and several views over the data are available. These can be accessed in the following ways:
-
-#### Spatial aggregates over a layer
+### NEX-GDDP and Loca datasets
 
-> Access spatial aggregates over the data by listing all dataset data for a particular year:
+> Example of a query for a NEX-GDDP/Loca dataset, providing lat and lon as query parameters:
 
 ```shell
-curl -i -XGET 'https://api.resourcewatch.org/v1/query/b99c5f5e-00c6-452e-877c-ced2b9f0b393?sql=SELECT * from nexgddp-historical-ACCESS1_0-prmaxday where year = 1960'
+curl -X GET 'http://api.resourcewatch.org/v1/query/aaadd6c3-93ea-44bc-ba8b-7af3f40d39e1?sql=SELECT * FROM data&lat=30&lon=9'
 ```
 
-> Access particular aggregates:
+> Example of a query for a NEX-GDDP/Loca dataset, providing a geostore id:
 
 ```shell
-curl -i -XGET 'https://api.resourcewatch.org/v1/query/b99c5f5e-00c6-452e-877c-ced2b9f0b393?sql=SELECT avg, min from nexgddp-historical-ACCESS1_0-prmaxday where year = 1960'
+curl -X GET 'http://api.resourcewatch.org/v1/query/aaadd6c3-93ea-44bc-ba8b-7af3f40d39e1?sql=SELECT * FROM data&geostore=972c24e1da2c2baacc7572ee9501abdc'
```

Comment:
all example urls should use https

Reply:
Replaced all links using http with https :ok_hand: